modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-store-info.adoc

```diff
@@ -350,3 +352,53 @@ The store formats are:
 | Relationship groups
 | `2^50` (1 Quadrillion)
 |===
+
+[[neo4j-admin-store-block]]
+=== Block
+
+.Store versions -- `block`
+[options="header"]
+|===
+| Store Format Name | Store Format Version | Introduced in | Unsupported from
+
+| `BLOCK_V1`
```

Comment:
I am a bit confused with the versioning. So, `BLOCK_V1` is introduced in 5.14 and is still supported, whereas `BLOCK_V2B` (beta) is introduced in 5.10 and is unsupported from 5.14. Is that correct?

Reply:
Yes that is correct. I'm not sure if we should mention the beta format at all (question for @mauvo). Users may encounter it if they use block format on neo4j 5.10-5.14.